.github/workflows/format.yml

```diff
@@ -7,10 +7,7 @@ on:
   push:
     branches-ignore:
       - 'dependabot/**'  # Avoid duplicates: only run the PR, not the push
-  pull_request_target:
-  workflow_dispatch:
-permissions:
-  contents: write
```

Comment:
I removed `pull_request_target` from here because it's pointless. It runs on the target repo, and checks out the base of the PR (not the PR itself), which is always already formatted anyway.

('workflow_dispatch' we should never have to use for this, and 'permissions' are redundant)